src/green_recognition/include/CircleDetection.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: June 15, 2017
+ * Description: Determines whether or not green is seen on
+ *              the screen then publishes a recommended twist
+ *              message.
+ * Usage:
+ * Subscribes to: camera // TODO: Determine topic names
+ * Publishes to:
```

Comment:
I'm OK if we just leave the `Subscribes to` and `Publishes to` sections out entirely. They'll probably be hard to maintain and likely get out of sync with the actual code fairly quickly.